lib/working_hours/computation.rb

```diff
@@ -85,6 +85,26 @@ def advance_to_working_time time, config: nil
       end
     end
 
+    def advance_to_closing_time time, config: nil
+      config ||= wh_config
+      time = in_config_zone(time, config: config).round
+      loop do
+        # skip holidays and weekends
+        while not working_day?(time, config: config)
+          time = (time + 1.day).beginning_of_day
+        end
+        # find next working range after time
+        time_in_day = time.seconds_since_midnight
+        time = time.beginning_of_day
```

Comment:
This won't work if there are several periods in a day, ex: `mon: {'08:00' => '12:00', '13:00' => '18:00'}`